primitives/npos-elections/src/lib.rs

```diff
@@ -283,7 +283,7 @@ impl<AccountId: IdentifierT> Voter<AccountId> {
 		})
 	}
 
-	/// Same as [`try_normalize`] but the normalization is only limited between elected edges.
+	/// Same as [`Self::try_normalize`] but the normalization is only limited between elected edges.
```

Comment:
Above in the same file you aded the parentheses to a function call. Are those even necessary?

```suggestion
	/// Same as [`Self::try_normalize()`] but the normalization is only limited between elected edges.
```

Reply:
they are necessary when there is ambiguity. I used it when it conflicts. Otherwise I have no opinion on other situation

Reply:
I think there is a way to explicitly tell docs that you want a function linked.

Reply:
ah yes we can prefix with `fn@...` but rust doc will render it with parenthesis anyway it seems.